### DISCUSSION TITLE
theme/colors coherence between spectrum implementations

### DISCUSSION THREAD

**Discussion by alexandremottet** [adobe/react-spectrum#1863] [Q&A] [answered]
Hi,

The Spectrum specification (https://spectrum.adobe.com/page/color/) specifies four different color scheme : `lighest`, `light`, `dark` and `darkest`.
If I take a look at the different implementation, both the CSS implementation (https://opensource.adobe.com/spectrum-css/get-started.html) and the web-components implementation (https://opensource.adobe.com/spectrum-web-components/components/theme) support `lightest` and `darkest` theme selection.
But the react implementation only offer two color scheme `light` and `dark`. In the documentation (https://react-spectrum.adobe.com/react-spectrum/theming.html), it is write `The darkTheme uses the Spectrum dark and darkest color themes`.

I was wondering why the react implementation is made that way and why it does not follow the Spectrum specification ?
If I miss something please let me know how I can override those values.
It is not helping the work between UX/UI designer and developers who can different technologies using Spectrum.

Thanks.

**Comment by snowystinger** [comment, marked as answer]
I think you're asking how you can use darkTheme. In order to use the darkTheme, you would change this line in your application
`import {defaultTheme} from '@adobe/react-spectrum';`
to
`import {darkTheme} from '@adobe/react-spectrum';`

If you need to lock it so that only the 'darkest' is being used. You can also supply `colorScheme="dark"` to your Provider. This will stop it from picking up OS settings and changing to the 'dark' theme. You will always be in 'darkest' because `darkTheme` maps to OS settings like this 'OS Light -> theme dark', 'OS Dark -> theme darkest'.

**Reply by alexandremottet** [reply]
Ok, I understand now.
So : 
- the theme `dark` and the color scheme `dark` will use Spectrum `darkest`
- the theme `dark` and the color scheme `ligth` will use Spectrum `dark`

Thank you so much
